packages/web-components/src/components/va-search-input/va-search-input.tsx

```diff
@@ -402,6 +428,77 @@ export class VaSearchInput {
     const type = this.el.getAttribute('type') || 'text';
     /* eslint-enable i18next/no-literal-string */
 
+    if (uswds) {
+      const formClasses = classnames({
+        'usa-search': true,
+        'usa-search--big': big && !small,
+        'usa-search--small': small && !big,
+      });
+      return (
+        <Host onBlur={handleBlur}>
+          <form class={formClasses} role="search">
+            <label class="usa-sr-only" htmlFor="search-field">Search</label>
+            <input 
+              class="usa-input" 
+              id="search-field" 
+              name="search"
+              type="search"
+              ref={el => (this.inputRef = el as HTMLInputElement)}
+              aria-autocomplete={ariaAutoComplete}
+              aria-controls={ariaControls}
+              aria-expanded={ariaExpanded}
+              aria-haspopup={ariaHasPopup}
+              aria-label={label}
+              autocomplete="off"
+              onFocus={handleInputFocus}
+              onInput={handleInput}
+              onKeyDown={handleInputKeyDown}
+              role={role}
+              value={value}
+            />
+            <button 
+              class="usa-button" 
+              type="submit"
+              onClick={handleButtonClick}
+            >
+              {!small && <span class="usa-search__submit-text">Search</span>}
+              {small &&
+                <va-icon
+                  icon="search"
+                  size={3}
+                  srtext="Search"
+                />
+              }
+            </button>
+            {isListboxOpen && (
+              <ul
```

Comment:
The "type ahead" feature was part of the original component so I've integrated into the v3 variation too. This is what it looks like for each of the USWDS v3 display types:

**default**

![typeahead-default](https://github.com/department-of-veterans-affairs/component-library/assets/872479/5da627c8-faf1-4514-9fa4-27e9443fdaca)

**big**


![typeahead-big](https://github.com/department-of-veterans-affairs/component-library/assets/872479/daf94755-f615-45b4-a482-1cc47317722f)

**small**

![typeahead-small](https://github.com/department-of-veterans-affairs/component-library/assets/872479/b57d2ea4-5082-4616-9f97-f1587f735f66)